tests/unit/remote/gdrive/test_gdrive.py

```diff
@@ -0,0 +1,16 @@
+import mock
+from dvc.remote.gdrive import RemoteGDrive
+
+
+@mock.patch("dvc.remote.gdrive.RemoteGDrive.init_drive")
+def test_init_drive(repo):
```

Comment:
not sure what exactly do we test here?

Reply:
Initialization of `gdrive.path_info` with input `url`.

Available tests were left there from previous PR  to have confirmation that they can be run automatically without asking user for auth input, etc

Tests should be extended/implemented.

Reply:
Any meaningful gdrive test will test this code implicitly anyway. I think this test might be safely removed.